src/vstest.console/CommandLine/CommandLineOptions.cs

```diff
@@ -229,7 +229,16 @@ public bool ShouldCollectSourceInformation
             {
                 return IsDesignMode;
             }
-        }        
+        }
+
+        /// <summary>
+        /// The instance of Executor that is used in current instance of process.
+        /// </summary>
+        public Executor CurrentExecutor
```

Comment:
Don't like this. Singleton, maybe?
Should we have some ProcessorManager, that we use in the executor and also pass on?